_usage/command-line.md

```diff
@@ -6,14 +6,18 @@ nav_order: 1
 
 ## Using client tools to work with Babelfish
 
-Many people use SQL Server Management Studio (SSMS) on MS Windows to connect to Babelfish. 
+Most people use SQL Server Management Studio (SSMS) on Microsoft Windows to
+connect to Babelfish.
 
-However, there is an ever-growing community which prefers to use the command line to work with the server.
+However, there is an ever-growing community which prefers to use the command
+line to work with the server.
```

Comment:
Please, don't do this. Keep it dull and factual.
Other folks will disagree on the 'ever-growing (etc.)'

Reply:
You've also drawn a comparison between two dissimilar objects:  Windows users and command line users.  Windows users also have command line tools.  Active voice would be an improvement too, maybe something like:

You can use a graphical interface or a command line tool to access Babelfish functionality on Windows or Linux.  On Linux, use:
*
*
On Windows, use:
*
*